### ISSUE TITLE
Right-to-left languages break UI

### ISSUE THREAD

**Issue by eightyy8** [recloudstream/cloudstream#410]
### Steps to reproduce

1. Change to a right to left language (Hebrew, Arabic).
2. Look at Library and action buttons (eg Stream option).
3. The search is squashed to right, and the action buttons on the left (because it's on the right).

### Expected behavior

Components on ui should stay either bordered from other components, or stay in the same position.

### Actual behavior

Components clash or flip options.

### Cloudstream version and commit hash

4.0.0 1eaa462

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

**Comment by Blatzar** [comment]
I don't see anything wrong switching languages, please include a screenshot of what you think is wrong.

**Comment by eightyy8** [comment]
![Screenshot_20230303-225950_1.png](https://user-images.githubusercontent.com/64216434/222850732-3584440f-7292-4406-a59c-b3c2d9e5f25c.png)
![Screenshot_20230303-230018_1.png](https://user-images.githubusercontent.com/64216434/222850762-99cbf093-0e2c-43ae-8499-81b06b29ac2c.png)

**Comment by eightyy8** [comment]
Unless that's how it's meant to be?

**Comment by eightyy8** [comment]
![Screenshot_20230303-230612_1.png](https://user-images.githubusercontent.com/64216434/222851880-286ef90d-89d9-4d63-b379-df60b781856b.png)

**Comment by Blatzar** [comment]
Only thing wrong with the image is the overlapping icons top right in library, but that didn't show up for me in Arabic.
Android by default changes the entire ui direction with RTL locales, mirrored stuff is intentional.

**Comment by eightyy8** [comment]
> Only thing wrong with the image is the overlapping icons top right in library, but that didn't show up for me in Arabic.

Yeah it breaks with mine on Arabic, possibly add a border to stop that possibility?

> Android by default changes the entire ui direction with RTL locales, mirrored stuff is intentional. 

I see.

**Comment by MudssirAhmed** [comment]
I'm also getting the same issue in the Urdu language, Fixing this one.

**Comment by MudssirAhmed** [comment]
@LagradOst I just created a [PR](https://github.com/recloudstream/cloudstream/pull/443). Kindly check.
